### PR TITLE
create pre sign up lambda to verify user

### LINKED ISSUE #17: auto confirm user after signUp

### PR DESCRIPTION
**Pull request by ghost**
- add hook to triger lambda when user sign up
- add handler to verify user
- introduce serverless-pseudo-parameters

### CODE REVIEW

File: cognito/handler.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const autoConfirmUser = async (event, context) => {
```
Comment (petarblazevski) [inline cognito/handler.js]: I suggest this to be named differently. Two reasons why:

1. If we need another function for the Cognito, `handler` is already taken the name
2. The name of the fine does not indicate what is this function doing

I would switch the things around. The file should be called `autoConfirmUser.js` and the function should be `main`. We need to be consistent with the naming, and `main` is used in all functions

Comment (petarblazevski) [reply cognito/handler.js]: And if you do not need the `context`, there is no reason to have it in the function

Comment (ghost) [reply cognito/handler.js]: good points, Thanks @petarblazevski 
fixed

**Review by petarblazevski — APPROVED** [review]
Looks great, mate!

**Pull request merged**